### PR TITLE
default wrong trap state not moving to next screen

### PR DESCRIPTION
**Pull request by schowkekarats360**

### CODE REVIEW

File: assets/src/apps/authoring/components/AdaptivityEditor/AdaptivityEditor.tsx [hunk 1]
```diff
@@ -152,7 +152,7 @@ export const AdaptivityEditor: React.FC<AdaptivityEditorProps> = () => {
             onChange={(changes: any) => {
               handleActionChange(action, changes);
             }}
-            allowDelete={!currentRule.correct}
+            allowDelete={!currentRule.default}
```
Comment (bsparks) [inline assets/src/apps/authoring/components/AdaptivityEditor/AdaptivityEditor.tsx:155]: seems like maybe this should be only if it is the default correct rule that we can't delete, not the defaultWrong rule

Comment (schowkekarats360) [reply assets/src/apps/authoring/components/AdaptivityEditor/AdaptivityEditor.tsx:155]: ohkay.... but in smartsparrow I saw that the defaultwrong for this was not allowed to be deleted either... so I make this change... Should I remove this or do you want to take a look?

Comment (bsparks) [reply assets/src/apps/authoring/components/AdaptivityEditor/AdaptivityEditor.tsx:155]: hmm when I'm looking in a lesson I don't see that, maybe it is something specific with forceProgress, i'll check another lesson

Comment (bsparks) [reply assets/src/apps/authoring/components/AdaptivityEditor/AdaptivityEditor.tsx:155]: yeah, when I check the lesson in the ticket, I see that you can't delete that one, I can't see anything in the editor that lets you set that though, maybe it's a legacy sparrow thing?? not sure, but IF I click the + to add an action and pick go to another screen then it changes it from next to something else and then I can delete it. so I think we are good to have this, and actually I just checked, it should be any correct that have to have a navigation on there, so I think we can just revert this part

Comment (schowkekarats360) [reply assets/src/apps/authoring/components/AdaptivityEditor/AdaptivityEditor.tsx]: ohk... I reverted back this change.

**Review by bsparks — APPROVED** [review]

**Pull request merged**